src/modules/UI.js

```diff
@@ -0,0 +1,74 @@
+import ListTemplate from './Class.js';
+import localGet from './Store.js';
+
+const todoContent = document.getElementById('listItem');
+const todoTask = document.getElementById('input');
+const displayList = () => {
+  todoContent.replaceChildren();
+  localGet().forEach((item, id) => {
+    let isCompleted;
+    if (item.completed === true) {
+      isCompleted = 'checked';
+    }
+    todoContent.innerHTML += `
+      <div class="toDoItem">
+        <div class="added-checkbox">
+
+        <input class='item' id='check-${id}', "completed")' type='checkbox' ${item.Checked ? 'true' : 'false'} onChange='updateList(${id}, "completed")' ${isCompleted}>
+        <input onkeyup='updateList(${id})' type="text" class='findInput' id='input-${id}' value='${item.description}' />
+
+        </div>
+        <i class="delete" onclick='removeList(${item.index})' id='delete-${item.index}'>🗑️</i>
+      </div>
+        `;
+  });
+};
+
+const addList = (description, completed, index) => {
+  const listAdded = new ListTemplate(description, completed, index);
+  const x = localGet();
+  x.push(listAdded);
+  localStorage.setItem('listStorage', JSON.stringify(x));
+  todoTask.value = '';
+
+  displayList();
+};
```

Comment:
- [ ] At the moment, I am not able to add tasks as the functionality is not working properly. Please kindly ensure the add task functionality works as expected so that other functionalities can be properly tested.

Reply:
@AnselemOdims Can you help me to fix it?

Reply:
This is not yet fixed